### ISSUE TITLE
Invalid ASN.1 syntax error

### ISSUE THREAD

**Issue by lochotzke** [eerimoq/asn1tools#37]
I just started with ASN.1 and wanted to use asn1tools.
The attached file (concatenated according to [this](https://stash.campllc.org/projects/SCMS/repos/scms-asn/browse?at=refs%2Fheads%2Fmaster)) causes `asn1tools.parser.ParseError: Invalid ASN.1 syntax at line 287, column 68: 'EndEntityType ::= BIT STRING {app (0), enrol (1) } (SIZE (8)) (ALL >!<EXCEPT {})': Expected ")".`
If I just remove that line I get `asn1tools.parser.ParseError: Invalid ASN.1 syntax at line 1439, column 23: 'RecipientInfo >!<(WITH COMPONENTS {': Expected ")".`
[http://asn1-playground.oss.com/](http://asn1-playground.oss.com/) does not complain. Is the asn file not correct?
[scms.asn.txt](https://github.com/eerimoq/asn1tools/files/2479210/scms.asn.txt)

**Comment by eerimoq** [comment]
The asn file is probably correct as the playground accepts it. There are plenty of ASN.1 constructs that asn1tools does not yet support, and it seems you found one of them. I have not used WITH COMPONENT(S) myself and I'm not sure what it does. Feel free to help out adding support for this.

**Comment by eerimoq** [comment]
I think you can simply remove all `WITH COMPONENT(S)` constraints from the specification and still successfully encode and decode data. Something like this:

```
EncryptedIndividualPLV ::= SEQUENCE {
    version         Uint8 (1),
    laId            LaId,
    enc-plv         EncryptedData
}
```

**Comment by eerimoq** [comment]
I just released version 0.126.0 of asn1tools. It can parse `WITH COMPONENT`, which greatly reduces the number of manual modifications to scms.asn. I can successfully compile [scms.asn.txt](https://github.com/eerimoq/asn1tools/files/2485793/scms.asn.txt).

**Comment by lochotzke** [comment]
Thanks. It works. I discovered 4 more issues (#38, #39, #40, #41). Maybe you can point me in the right direction so that I might fix them.
